package.json

```diff
@@ -14,14 +14,15 @@
   },
   "dependencies": {
     "express": "4.16.2",
-    "should": "^8.4.0",
+    "read-pkg-up": "3.0.0",
+    "should": "8.4.0",
     "sinon": "1.17.7",
+    "stoppable": "1.0.6",
     "supertest": "3.0.0",
-    "stoppable": "boneskull/stoppable#boneskull-patch-1",
     "when": "3.7.8"
   },
   "peerDependencies": {
-    "node-red": "0.18.x"
+    "node-red": "~0.18.4"
```

Comment:
can we be more forgiving than this?

Reply:
since we depend on the internals of the runtime currently, better safe than sorry I thought.